tests/allure/src/main/java/org/mule/test/allure/AllureConstants.java

```diff
@@ -830,4 +830,17 @@
 
     }
   }
+
+  interface PolicyZeroDowntimeFeature {
+
+    String POLICY_ZERO_DOWNTIME = "Zero Downtime";
+
+    interface PolicyReorderStory {
```

Comment:
Put this inside `POLICY_DEPLOYMENT` feature and remove `PolicyZeroDowntimeFeature `.